components/blitz/src/ome/formats/importer/cli/CommandLineImporter.java

```diff
@@ -344,7 +346,7 @@ public static void usage() {
             + "  $ %s -s localhost -u username -w password -d 50 --debug ALL foo.tiff\n"
             + "\n"
             + "For additional information, see:\n"
-            + "http://www.openmicroscopy.org/site/support/omero5/users/command-line-import.html\n"
+            + "http://www.openmicroscopy.org/site/support/omero5.1/users/cli/import.html\n"
```

Comment:
After a discussion, I'm withdrawing my worries. Policy moving forward will be to include `major.minor` in the URLs and script the migration from `x.y` to `x.y+1`.